source/common/http/conn_manager_impl.cc

```diff
@@ -1136,6 +1138,7 @@ void ConnectionManagerImpl::ActiveStream::decodeData(Buffer::Instance& data, boo
                                connection_manager_.read_callbacks_->connection().dispatcher());
   filter_manager_.maybeEndDecode(end_stream);
   filter_manager_.streamInfo().addBytesReceived(data.length());
+  filter_manager_.streamInfo().getDownstreamBytesMeterer()->addBodyBytesReceived(data.length());
```

Comment:
This `data.length()` won't include any framing for DATA frames -- which it should and it doesn't seem that it's counting in H2 codec. Will point out in H2 codec where you should account for that.